src/extractApi/LinkedApisJsonObject.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export class LinkedApisJsonObject {
```

Comment:
dont need two types called LinkedApisJsonObject and SingleApiJsonObject. Can use a single Interface and name it as IExtractConfig. Can also move it to the same file as where the functionality is.